docs/quickstart.rst

```diff
@@ -3,361 +3,556 @@
 Quickstart
 ***************
 
-This tutorial gets you started with the Inmanta orchestration tool.
-
 Inmanta is intended to manage complex infrastructures, often in the cloud or other virtualized environments.
-In this guide, we go for a less complex setup: install the Drupal CMS on two VM-like containers.
-First, we use Docker to set up a basic environment with two empty VM-like containers, an Inmanta server and a postgres server used by inmanta as a database.
-Then, we use Inmanta to install Drupal on these VM-like containers.
+In this guide, we go for a less complex setup: Installing `containerlab <https://containerlab.dev/>`_ and configuring `SR Linux <https://learn.srlinux.dev/>`_ containers using **Inmanta service orchestrator** and ``gNMI``.
```

Comment:
```suggestion
In this guide we start simple and manage a 3 node CLOS network with spine and two leaf switches. In this guide we will install `containerlab <https://containerlab.dev/>`_ and configure `SR Linux <https://learn.srlinux.dev/>`_ containers using **Inmanta open source orchestrator** and ``gNMI``.
```

Reply:
`In this guide` is repeated here. it kind of sounds unnatural.  
Let's apply the suggestion here and then I'll rectify it later when everything is in order.

Reply:
yeah, the two times in this guide is a mistake.